dashboard/test/lib/contact_rollups_v2_test.rb

```diff
@@ -0,0 +1,76 @@
+require 'test_helper'
+require 'cdo/contact_rollups/v2/pardot'
+require 'cdo/log_collector'
+
+class ContactRollupsV2Test < ActiveSupport::TestCase
+  setup do
+    EmailPreference.delete_all
+    ContactRollupsPardotMemory.delete_all
+    ContactRollupsFinal.delete_all
+  end
+
+  teardown do
+    EmailPreference.delete_all
```

Comment:
This is to make sure each test cleans up after itself. We use direct SQL insertions and activerecord-import in the pipeline; they are not automatically rolled back when a test finishes.